determined-ai/determined
# style: responsive webui [DET-4417, DET-4420]

## Description

Update the WebUI to be mobile friendly. Key areas of updates:

- Primary navigation
- Support for Responsive tables
- Filter groups in Section headers

<img width="362" alt="Screen Shot 2020-10-28 at 11 49 00 AM" src="https://user-images.githubusercontent.com/220971/97476450-05aa0180-1914-11eb-9da8-0ea3c9a006b8.png">
<img width="362" alt="Screen Shot 2020-10-28 at 11 49 11 AM" src="https://user-images.githubusercontent.com/220971/97476456-0773c500-1914-11eb-8e2d-f4be989064f2.png">
<img width="362" alt="Screen Shot 2020-10-28 at 11 50 28 AM" src="https://user-images.githubusercontent.com/220971/97476460-08a4f200-1914-11eb-8b30-e0209e4a8356.png">
<img width="362" alt="Screen Shot 2020-10-28 at 11 50 52 AM" src="https://user-images.githubusercontent.com/220971/97477314-160eac00-1915-11eb-9e2f-0f7b12824dd5.png">
<img width="362" alt="Screen Shot 2020-10-28 at 11 50 52 AM" src="https://user-images.githubusercontent.com/220971/97476485-0fcc0000-1914-11eb-8f44-8d50d53012e8.png">
<img width="362" alt="Screen Shot 2020-10-28 at 11 51 11 AM" src="https://user-images.githubusercontent.com/220971/97476489-10fd2d00-1914-11eb-87db-d26167003351.png">





## Test Plan

- Check every WebUI page to ensure they render properly in desktop, tablet and mobile views (check to make sure they are fluid as well)
  - Login page
  - Dashboard
  - Experiment List
  - Experiment Detail
  - Trial Detail
  - Trial Logs
  - Task List
  - Cluster
  - Master Logs
- Check for the tables to render properly for mobile views.
- Check for the filter groups to render properly. (the filter options that appear across from the Section headers. For example the `User` or `State` filters on the experiment list page)




## Commentary (optional)




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234
